### ISSUE TITLE
XAML Designers claims "name AnimationBehavior does not exist...

### ISSUE THREAD

**Issue by jhwheuer** [XamlAnimatedGif/XamlAnimatedGif#90]
Hi Thomas, been using your component for months now.
All of a sudden (no changes in source, packages etc.), the XAML designer gives the following error:

> The name "AnimationBehavior" does not exist in the namespace "using:XamlAnimatedGif".

Any ideas?

**Comment by thomaslevesque** [comment]
Hi @jhwheuer,

The VS XAML designer is full of glitches so I see this error sometimes and it usually disappears on its own... not sure what's causing it. Did you install the library via NuGet?

**Comment by jhwheuer** [comment]
Yep, used NuGet. De-Installed, Re-Installed. De-Installed, rebooted, Re-Installed... 
keeps cropping up like a bad politician.
Need to take it out to complete the build... even when the designer is not open.... so there must be something a bit deeper? nasty. %-)

**Comment by jhwheuer** [comment]
Hi Thomas, I did not change any of the code. It just started not working...

The assembly is properly referenced... removing it twice and re-installing should have done that... otherwise, how else should I "Properly Do that?" %-)

On the packages... there is no packages directory in the sln. Do you mean the bin dir?

**Comment by thomaslevesque** [comment]
> On the packages... there is no packages directory in the sln. Do you mean the bin dir?

No, I mean `packages` at the solution level... For classic projects (WPF, WinForms, Console, etc targeting the full .NET framework), this is where the NuGet packages are. But I think it's different for .NET Core or UWP, the packages are in a shared location.

**Comment by jhwheuer** [comment]
Yep, it is... 
and that would not solve the problem. NuGet'ed the local cache, the works. nada.

Then I found this suggestion: http://stackoverflow.com/questions/28216096/wpf-the-name-does-not-exist-in-the-namespace

Changed from x64 to x86 and back again, AND EVERYTHING IS FINE AGAIN.

You gotta love dem IDEs. This kind of wasting-8-hours-of-my-dev-time stuff is why I said goodbye to Eclipse after a decade...

THANKS A BUNCH!

PS. Ever thought about a "RAM-CACHE" option like I suggested a while back? So keep a local copy of the images in RAM rather than on disk.... Should be possibe to manage to a certain cache size... Working on Desktop, so memory is not really a concern.
